de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/IHUPackingMaterialsCollector.java

```diff
@@ -155,4 +155,8 @@ void addLU(I_M_HU luHU,
 	 */
 	IHUPackingMaterialsCollector<T> setProductIdSortComparator(Comparator<Integer> productIdsSortComparator);
 
+	void setisCollectTUNumberPerOrigin(boolean b);
+
+	void setisCollectAggregatedHUs(boolean b);
```

Comment:
* same about method naming
* also about those two methods:
  * when does it make sense to call `setisCollectAggregatedHUs(false)` (->javadoc)
  * or maybe we don't need them at all?